### ISSUE TITLE
Unable to start remote tunnel using vscode cli

### ISSUE THREAD

**Issue by saurabh-c-rai** [microsoft/vscode-remote-release#9509]
Steps To Reproduce : -

1. Login to a remote machine via ssh and switch to sudo user via `sudo su`.
2. Download the cli using the curl request - 
    `curl -Lk 'https://update.code.visualstudio.com/1.85.2/cli-linux-x64/stable' --output vscode_cli.tar.gz`
3. Unzip the archive using the command -
    `tar -xf vscode_cli.tar.gz`
4. Start the remote tunnel using -
    `./code tunnel`

Expected Results :-
Remote tunnel should start.

Actual Result :-
Remote tunnel doesn't start.
Getting below error message.

```
./code tunnel
./code: /lib64/libc.so.6: version `GLIBC_2.29' not found (required by ./code)
./code: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by ./code)
./code: /lib64/libc.so.6: version `GLIBC_2.25' not found (required by ./code)
./code: /lib64/libc.so.6: version `GLIBC_2.18' not found (required by ./code)
./code: /lib64/libc.so.6: version `GLIBC_2.27' not found (required by ./code)
``` 

The exact same steps were working previously and are not working now.

PS. My remote is currently using Centos 7 and we are not in position to upgrade it. Hence, cannot use the latest cli and vscode.

**Comment by connor4312** [comment]
You can use the "alpine" build of the CLI that does not have a dependency on GLIBC. Or, you can update your OS.

**Comment by saurabh-c-rai** [comment]
@connor4312 , As i mentioned, we cannot upgrade OS right now due to production dependency.
1. can you share any link for downloading the alpine build for vscode CLI. I cannot find any link for alpine builds.
2. Another question is since the glibc dependency was released in v1.86, why is the issue occuring in v1.85.2.

**Comment by connor4312** [comment]
Yep, here

https://update.code.visualstudio.com/latest/cli-alpine-x64/insider

> Another question is since the glibc dependency was released in v1.86, why is the issue occuring in v1.85.2.

The CLI always depended on a newer version of glibc, it was not actually synchronized to what the VS Code server needed until last released. It just happened that that release also included a glibc bump
